src/helpers/assets.js

```diff
@@ -81,6 +80,16 @@ const addEthPlaceholder = (
   return assets;
 };
 
+const getTotal = assets =>
+  reduce(
+    assets,
+    (acc, asset) => {
+      const balance = asset?.native?.balance?.amount ?? 0;
+      return add(acc, balance);
+    },
+    0
+  );
+
 export const buildCoinsList = (
```

Comment:
This file will be easier to review by following along on the comments in `buildCoinsList` instead of looking at the diffs